pkg/controller/middleware/currentpage.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
```

Comment:
I think this already exists in the templatevars middleware?

Reply:
@whaught https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/controller/middleware/template.go#L44

Reply:
I was going to replace that. This gives you the option to get the path, page, or full url. Slightly nicer than the stringContains?